macros/fivetran/base/fivetran_fb_ad_creatives.sql

```diff
@@ -14,7 +14,7 @@ with base as (
         coalesce(page_link,template_page_link) as url,
         coalesce(page_link,template_page_link) as base_url,
         '/' || {{ dbt_utils.get_url_path('coalesce(page_link,template_page_link)') }} as url_path,
-        row_number() over (partition by creative_id, page_link order by _FIVETRAN_SYNCED desc) as row_num
+        row_number() over (partition by creative_id, page_link order by _fivetran_synced desc) as row_num
```

Comment:
I wonder if this will break Snowflake. I don't think it will 🤞

Reply:
Honestly, I just went ahead and switched it back. Figured there might be some reason this field is always uppercased.